src/cluster/placement/algo/sharded_test.go

```diff
@@ -1306,6 +1306,42 @@ func TestBalanceShardsForShardedWhenImbalanced(t *testing.T) {
 	assert.Equal(t, expectedInstances, balancedPlacement.Instances())
 }
 
+func TestInitialPlacementIsBalanced(t *testing.T) {
+	instances := make([]placement.Instance, 0)
+
+	for i := 0; i < 10; i++ {
+		instances = append(instances,
+			placement.NewEmptyInstance(fmt.Sprintf("instance-0-%03d", i), "iso0", "zone", "endpoint", 1),
+			placement.NewEmptyInstance(fmt.Sprintf("instance-1-%03d", i), "iso1", "zone", "endpoint", 1),
+		)
+	}
+
+	ids := make([]uint32, 1024)
+	for i := range ids {
+		ids[i] = uint32(i)
+	}
```

Comment:
From this code I can't get what those ids are. Are they shard ids? Instance ids? Some other ids? Perhaps some naming would help here.

Reply:
`ids` => `shardIDs`
 https://github.com/m3db/m3/pull/4020/commits/1721594d34f54216415e58e57a196e1e00e149d2#diff-05cd0fd1dc4c1ee3969a247ba1ce3fb2fb21e609a49c08e470f0df5af02e88c5R1319-R1321